Flutter/lib/NameScreen.dart

```diff
@@ -33,26 +42,15 @@ class NameScreen extends StatelessWidget {
     final nextButton = SizedBox(
       width: width,
       height: height,
-      child: RaisedButton(
-        shape: RoundedRectangleBorder(
-          borderRadius: BorderRadius.circular(25.0),
-          side: BorderSide(color: Colors.white)),
-        color: Colors.blue,
-        child: Icon(
-          Icons.arrow_forward_outlined,
-          color: Colors.white,),
-        onPressed: () {
-          if (_formKey.currentState.validate()) {
-            _navigate(myControler.text);
-          }
-        }
-      ),
+      child: AppTheme.nextButton(context, () {
+        if (_formKey.currentState.validate()) _navigate(myControler.text);
+      }),
     );
 
     return Container(
       decoration: BoxDecoration(
-        image: DecorationImage(
-          image: AssetImage('assets/background.png'), fit: BoxFit.cover)),
+          image: DecorationImage(
+              image: AssetImage(getImage()), fit: BoxFit.cover)),
```

Comment:
This should be called `getBackground()`. `getImage()` is confusing

Reply:
Resolved in commit `app-style feaf5a9`